docs/release-notes/multi-arch-and-arm64.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Command: Produce arm64 packages for Linux and Darwin.
+-  Images: Produce multi-arch image manifests with amd64, arm64, and ppc64 architectures.
```

Comment:
in release notes we usually refer to "Images" as in "environment images", not the docker images for agent and master. also I am not sure what do you mean by command. perhaps:

```suggestion
-  Installation: 
    - added deb, rpm, and homebrew packages for arm64.
    - added master and agent docker images for arm64 and ppc64.
```

Reply:
For the categories I was just trying to follow https://github.com/determined-ai/determined/tree/master/docs/release-notes.

You are right in that "Command" doesn't really make sense here so it was removed. However, "added deb, rpm, and homebrew packages for arm64" is more indicative of the documentation for your homebrew PR, there is no change in this PR regarding that behavior.

Also this PR does not allow "Installation" in the sense that a user might expect, there is further work in other areas needed to accomplish that task. When we refer to the images it should be as "container images" rather than the name-brand "docker".

Reply:
I'd still suggest being more specific these are only master & agent images, not all of them.